rusk/Cargo.toml

```diff
@@ -66,7 +66,6 @@ dusk-pki = "0.8"
 phoenix-core = "0.14.0-rc"
 anyhow = "1.0"
 rand = "0.8"
-bid-circuits = { path = "../circuits/bid" }
 blindbid-circuits = { path = "../circuits/blindbid" }
```

Comment:
I think you should remove also the `blindbid`, it should be used only by the bid contract (double check to be sure).